src/thunderbots/software/ai/intent/direct_wheels_intent.h

```diff
@@ -0,0 +1,52 @@
+#pragma once
+
+#include "ai/intent/intent.h"
+#include "ai/primitive/direct_wheels_primitive.h"
+
+class DirectWheelsIntent : public Intent, public DirectWheelsPrimitive
+{
+   public:
+    static const std::string INTENT_NAME;
+    // Power is a fraction of the total power we can apply to the robots,
+    // with +-255 being the max/min, and 0 being no power.
```

Comment:
I think this probably belongs in the full description for the constructor, ie:
```
    /**
     * Creates a new DirectWheels Intent
     *   
     * Power is a fraction of the total power we can apply to the robots, with 
     * +-255 being the max/min, and 0 being no power.
     *
     * @param robot_id the id of the robot
     * @param front_left_wheel_power a value between -255 and 255, where positive is
     * clockwise
     * @param back_left_wheel_power a value between -255 and 255, where positive is
     * clockwise
     * @param front_right_wheel_power a value between -255 and 255, where positive is
     * clockwise
     * @param back_right_wheel_power a value between -255 and 255, where positive is
     * clockwise
     * @param dribbler_rpm the dribbler rpm
     * @param priority The priority of this Intent. A larger number indicates a higher
     * priority
     */

```

Reply:
fixed